pkgs/development/libraries/twolame/default.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, fetchFromGitHub
+, autoreconfHook, pkgconfig, libsndfile }:
+
+stdenv.mkDerivation rec {
+
+  name = "twolame-${version}";
+  version = "git-2017-09-27";
```

Comment:
https://nixos.org/nixpkgs/manual/#sec-package-naming

> The version part of the name attribute must start with a digit (following a dash) — e.g., "hello-0.3.1rc2".
> If a package is not a release but a commit from a repository, then the version part of the name must be the date of that (fetched) commit. The date must be in "YYYY-MM-DD" format. Also append "unstable" to the name - e.g., "pkgname-unstable-2014-09-23".

Reply:
But don't append unstable, since this is the only version and you treat it as stable.

Reply:
In sum: erasing the `git-` part?

Reply:
Yes. (And from the commit message too.)